Evergreen/Manifests/Inkscape.json

```diff
@@ -0,0 +1,51 @@
+﻿{
+    "Name": "Inkscape",
+    "Source": "https://inkscape.org/",
+    "Get": {
+        "Uri": "https://inkscape.org/release",
```

Comment:
The approach with Evergreen is to not scrape HTML. When this page changes, there's a good chance the function will break